app/middlewares/user.js

```diff
@@ -0,0 +1,27 @@
+const { check } = require('express-validator');
+
+const userConfig = require('../../config').common.user;
+
+const checkAttributesMissing = value => {
```

Comment:
Check that express-validator has a "exists" function. You can reject this with withMessage to explain that attribute is missing (remember that you are an API, your errors must be more specific for the clients)